### DISCUSSION TITLE
Emmet support

### DISCUSSION THREAD

**Discussion by Mr0Bread** [zed-industries/zed#6587] [Feature Requests]
Does Zed support emmet?

**Comment by zakariabinsaifullah** [comment]
Right now, maybe Emmet is not supported yet. But, it is an essential feature and I hope this feature will be added in short.

**Comment by JosephTLyons** [comment]
Just wanted to chime in and say that we are going to be putting some focus on improving the experience for web devs in Zed, which means we are likely going to prioritize adding Emmet support here soon :)

**Reply by airamstudio** [reply]
any news on this? i need emmet lol im so used to it its a dealbreaker

**Reply by Moshyfawn** [reply]
> any news on this? i need emmet lol im so used to it its a dealbreaker

https://github.com/zed-industries/zed/issues/4992
